README.md

```diff
@@ -166,33 +166,35 @@ now, when you add the files it ignores node_modules directory in your project.
 
   These steps are to be followed when your forked repository is few commits behind original repository
 
-```sh
-
-  - Add original repository url here 
-
-    git remote add upstream https://github.com/whoever/whatever.git
-
-  - Fetch all branches of the repository
-
-    git fetch upstream
-
-  - Make sure you are on master branch
-
-    git checkout master
-
-  -  Rewrite your master branch so that any commits of yours that aren not already in upstream/master are replayed on top of that other branch
-
-    git rebase upstream/master
-
-    This should update your forked repository with original repository
-
-    However if you do not want to rewrite history of master branch then replace last command with this
-
-    git merge upstream/master
-    
- 
-
-```
+  1. Check if you have the forked repository added to your remotes.
+  ```
+  git remote -v
+  ```
+  If you see the forked repository listed in your remotes you can skip to point 3.
+
+  2. Add the forked repository as a remote
+  ```
+  git remote add upstream https://github.com/whoever/whatever.git
+  ```
+  3. Fetch changes from forked repository
+  ```
+  git fetch upstream
+  ```
+  Any new changes and branches from the original forked repository should now be fetched to your local repository. You can now choose to either rebase your local branch, or merge the changes from the forked repository into your branch.
+
+  If you are unsure about which option to pick, you can read more about [the differences between rebasing and merging in this article](https://www.atlassian.com/git/tutorials/merging-vs-rebasing).
+
+  4. Rebase your local branch, or merge the changes
+
+  **Rebase**
+  ```
+  git rebase upstream/master
+  ```
+
+  **Merge**
+  ```
+  git merge upstream/master
```

Comment:
No need to merge after rebase

Reply:
True, however the step states to either rebase or merge - but maybe that can be written a bit more clearer.

Reply:
Agreed

Reply:
Moved some text around and rephrased it a bit - should be more clear now.